go/air/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.12.8
+
+WORKDIR /go/src/app
+COPY main.go go.mod ./
+
+ENV GO111MODULE=on
```

Comment:
no need :stuck_out_tongue: 

could you copy the `Dockerfile` in `go/grouter` ?
I'll try to make uniform `Dockerfile`s for all frameworks